source/Dialog.h

```diff
@@ -132,7 +136,9 @@ Dialog::Dialog(T *t, void (T::*fun)(int), const std::string &text, int initialVa
 template <class T>
 Dialog::Dialog(T *t, void (T::*fun)(const std::string &), const std::string &text,
 	std::string initialValue, Truncate truncate)
-	: stringFun(std::bind(fun, t, std::placeholders::_1)), input(initialValue)
+	: stringFun(std::bind(fun, t, std::placeholders::_1)),
+	allowsFastForward(true),
```

Comment:
Why only on this dialog? Why not allow it on all dialogs?

Or if we want to restrict this specifically to the search dialog, then I feel we should make this more specific than allowing fast forward on all text-input dialogs.

Reply:
> Why only on this dialog? Why not allow it on all dialogs?

Some dialogs can appear in situations where you do want fast forward to be interrupted. For example, an `on enter` can be given a dialog, and that dialog may be pointing out something important that you want to slow down for (e.g. perhaps there's a heavy enemy presence in the system that you're being made note of).

> Or if we want to restrict this specifically to the search dialog, then I feel we should make this more specific than allowing fast forward on all text-input dialogs.

What other text-input dialogs are used? Naming ships, I presume (which pop up in the shipyard, which doesn't allow fast forward when interrupt is active, so whether or not the naming ships dialog allows fast forward or not is inconsequential), but what else?

Reply:
I fully agree to the intent of this PR; the search dialog in the map should not interrupt fast-forward, because the map-panel also doesn't interrupt fast-forward.
But allowing fast-forward should not be some hidden side-effect based on which constructor for Dialog was chosen; it should be very explicit.

Reply:
Is the latest commit more what you're thinking of?

Reply:
Yes, the last commit handles my comment. Thanks!